media/js/mozorg/home/home-b.js

```diff
@@ -32,6 +54,11 @@
         $('#fx-download-link').addClass('hidden');
     }
 
+    // test for matchMedia
+    if ('matchMedia' in window) {
```

Comment:
Could this block be inside the document ready handler below? `mqIsTablet` doesn't seem to be used anywhere else (the var could be defined there too?).